laravel/resources/views/inc/banners/banner-config-input.blade.php

```diff
@@ -1,78 +1,80 @@
-<div class="col-md-12 row">
+<div class="row" id="optionInputGroup">
     @if (isset($configuration))
-    <input type="hidden" name="configuration[banner_configuration_id][]" value="{{ $configuration->id }}">
+        <input type="hidden" name="configuration[banner_configuration_id][]" value="{{ $configuration->id }}">
     @endif
-
-    <div class="col-md-4">
+    <div class="col-lg-12 mb-3">
+        <div class="row">
+            <div class="col-lg-2">
+                @if (isset($configuration))
+                <a class="form-control btn btn-danger btn-sm" href="{{ route('banner.template.configuration.delete', ['banner_configuration_id' => $configuration->id]) }}">Delete</a>
+                @else
+                    <button type="button" class="form-control btn btn-danger btn-sm" id="remove-config-row">Remove row</button>
+                @endif
+            </div>
+        </div>
+    </div>
+    <div class="col-lg-4 mb-2">
         <label for="validationXCoordinate" class="form-label">X-Coordinate (horizontal)</label>
-        <input class="form-control" id="validationXCoordinate" type="number" min="0" step="1" max="{{ $template->width }}" name="configuration[x_coordinate][]" value="{{ old('configuration[x_coordinate][]', (isset($configuration)) ? $configuration->x_coordinate : '') }}" placeholder="30" aria-describedby="xcoordinateHelp" required>
+        <input class="form-control" id="validationXCoordinate" type="number" min="0" step="1" max="{{ $template->width }}" name="configuration[x_coordinate][]" value="{{ old('configuration[x_coordinate][]', (isset($configuration)) ? $configuration->x_coordinate : '') }}" placeholder="e.g. 30" required>
```

Comment:
the implementation seems like false.
https://getbootstrap.com/docs/5.3/forms/validation/#server-side

> For invalid fields, ensure that the invalid feedback/error message is associated with the relevant form field using 
> aria- describedby (noting that this attribute allows more than one id to be referenced, in case the field already points to 
> additional form text).

but yupp it was my fault to delete it and will be changed.

Reply:
Note:
i found in other files sometimes `aria-describedby` is used and sometimes its not used. So i will fix all affected files with form vaildations. I remind why i delete this. In my level of knowledge `aria-describedby` is for Screenreaders.

https://www.w3.org/WAI/standards-guidelines/aria/
> WAI-ARIA, the Accessible Rich Internet Applications Suite, defines a way to make Web content and Web applications more accessible to people with disabilities. It especially helps with dynamic content and advanced user interface controls developed with HTML, JavaScript, and related technologies.

So, learned something again :-)